packages/charts/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.base.tsx

```diff
@@ -487,12 +487,18 @@ export class HorizontalBarChartWithAxisBase
     const xDomain = [Math.min(this.X_ORIGIN, xMin), Math.max(this.X_ORIGIN, xMax)];
     if (isNumericScale) {
       const yMax = d3Max(this._points, (point: IHorizontalBarChartWithAxisDataPoint) => point.y as number)!;
+      const yMin = d3Min(this._points, (point: IHorizontalBarChartWithAxisDataPoint) => point.y as number)!;
+      const yDomainMax = Math.max(yMax, this.props.yMaxValue || 0);
+      const yMinProp = this.props.yMinValue || 0;
+      const yDomainMin = yMin < yMinProp ? Math.min(0, yMin) : yMinProp;
+
+      const yDomainPadding = Math.abs((yMax - yMin) * 0.1);
```

Comment:
why is this padding needed?

Reply:
This issue is similar to the scatter chart where, if the bar has y=yMin(=0 in positive cases) it overlaps with the axis.

Reply:
We can reduce the scale range by subtracting the bar height, similar to how we adjust the x-axis in vertical bar charts.

Reply:
I tried but that did not work out of box. Probably the solution it is not available for y scale.